packages/python-google-compute-engine/google_compute_engine/distro_lib/helpers.py

```diff
@@ -98,3 +124,18 @@ def CallNtpdate(logger):
     logger.warning('Failed to sync system time with ntp server.')
   else:
     logger.info('Synced system time with ntp server.')
+
+def CallWriteViaSysCtl(logger, name, value):
+  """Write a variable using sysctl,
+  Args:
+      name: name of the sysctl variable.
```

Comment:
Please include variable type (like other functions do). Also, you missed the logger variable.

Reply:
added logger, value can be a string or int. So, it does not seem worth the confusion to provide a type here.